README.md

```diff
@@ -2678,77 +2678,120 @@ It’s not exhaustive, as titles and details vary.
+     - Joe Wallin,
+      [*RSUs vs. Restricted Stock vs. Stock Options*](http://joewallin.com/2014/09/13/rsus-vs-restricted-stock-vs-stock-options/), 2014\.
+    - **Taxes**
+      - Mark P. Cussen, Investopedia,[*How Restricted Stock And RSUs Are Taxed*](http://www.investopedia.com/articles/tax/09/restricted-stock-tax.asp?performancelayout=true).
+      - Barry Kramer,
+        [🔑*The Tax Law that is (Unintentionally) Hammering Silicon Valley Employees*](https://medium.com/@barryjk/the-tax-law-that-is-unintentionally-hammering-silicon-valley-employees-894a7b54ba8a),
+        2015\.
+      - Kaye A. Thomas, Fairmark,
+        [*AMT and Long-Term Capital Gain*](http://fairmark.com/general-taxation/alternative-minimum-tax/amt-long-term-capital-gain/),
+        2014\.
+      - Robert W. Wood, Forbes,
+        [*Ten Tax Tips For Stock Options*](http://www.forbes.com/2010/03/10/10-tax-tips-stock-options-personal-finance-robert-wood.html),
+        2010\.
+      - NCEO,
+        [*Stock Options and the Alternative Minimum Tax (AMT)*](https://www.nceo.org/articles/stock-options-alternative-minimum-tax-amt)
+    - **Specialized topics**
+      - Steven Ayre, Accelerated Vesting,
+        [*What Is An 83(b) Election and When Do I Make It?*](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/),
+        2013\.
+      - Joshua Levy and Joe Wallin, The Startup Law Blog,
+        [*The Problem With Immediately Exercisable ISOs*](http://thestartuplawblog.com/the-problem-with-immediately-exercisable-isos/),
+        2015\.
+      - Tahir J. Naim, Fenwick,
+        [*Section 409A Valuations and Stock Option Grants for Start-up Technology and Life Science Companies*](http://www.fenwick.com/FenwickDocuments/409_Valuations_Stock_Options.pdf).
+      - Babak Nivi, Venture Hacks,
+        [*How to make a cap table*](http://venturehacks.com/articles/cap-table), 2007\.
+      - Mary Russell, Stock Option Counsel,
+      [*Can the Company Take Back My Vested Shares?*](http://stockoptioncounsel.com/blog/standards-ownership-canthecomanytakebackmyvestedshares), 2017\.
+      - Mary Russell, Stock Option Counsel,
+      [*Early Expiration of Startup Stock Options - Part 2 -The Full 10-Year Term Solution*](http://stockoptioncounsel.com/blog/early-expiration-of-startup-stock-options-part-2-the-full-10-year-term-solution/2017/3/28),
+      2017\.
+      - Mary Russell, Stock Option Counsel,
+      [*Early Expiration of Startup Stock Options - Part 3 - Examples of Good Equity Design by Company Stage*](http://stockoptioncounsel.com/blog/early-expiration-of-startup-stock-options-part-3-examples-of-good-startup-equity-design-by-company-stage/2017/8/11),
+      2017\.
+      - Dan Shapiro, [*Vesting is a hack*](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/), 2012\.
+      - Scott Edward Walker, VentureBeat,
```

Comment:
This one is founders too.

Reply:
Moved. (To be clear I moved the Dan Shapiro piece since it's the only complete citation in the lines above.)